packages/localize/CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+### Fixed
+
+- Removed accidental export of `_msg` (use `msg` instead).
```

Comment:
Do you want to add a mention of `msg()` being a no-op by default?

Reply:
There's no change here, it's always been this way. There just used to be an extra export of the same thing called `_msg`.